llvm/utils/TableGen/X86DisassemblerTables.cpp

```diff
@@ -885,7 +887,10 @@ void DisassemblerTables::emitContextTable(raw_ostream &o, unsigned &i) const {
   for (unsigned index = 0; index < ATTR_max; ++index) {
     o.indent(i * 2);
 
-    if ((index & ATTR_EVEX) || (index & ATTR_VEX) || (index & ATTR_VEXL)) {
+    if ((index & ATTR_EVEX) && (index & ATTR_64BIT) && (index & ATTR_OPSIZE) &&
```

Comment:
Could we remove the `(index & ATTR_64BIT)`? From the name `IC_EVEX_OPSIZE_ADSIZE`, it should not check this.

Reply:
Do we still need this?

Reply:
ATTR_64BIT is not needed